CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ The prerequisites for building, testing, and deploying from this repository are:
 
 * The [.NET SDK](https://get.dot.net/).
 * The [Rust toolset](https://www.rust-lang.org/tools/install)
+* uniffi-bindgen-cs which you can install with the following command: `cargo install uniffi-bindgen-cs --git https://github.com/NordSecurity/uniffi-bindgen-cs --tag v0.7.0+v0.25.0`
```

Comment:
Is this necessary to document, given the build script prescribed later in this file runs this `cargo install` script?

Reply:
It looks like the param -InstallPrerequisites needs to be passed into `generate_cs_bindings.ps1` but `build_all.ps1` never passes that. I wonder if `generate_cs_bindings.ps1` could just check if `uniffi-bindgen-cs` is in the path rather than take a param and run the cargo install command if it is not there. I will look into making this change and then I agree this line could be removed.

Reply:
I have worked out how to update the script. I will create a new PR for that script update. I've removed this line.